README.md

```diff
@@ -52,18 +53,27 @@ pod install
 
 ### [Swift Package Manager](https://github.com/apple/swift-package-manager)
 
-**Note**: This feature is only available with Swift 5.3 (Xcode 12) or later.
+> [!NOTE]
+> This feature is only available with Swift 5.3 (Xcode 12) or later.
```

Comment:
I don't think the note is required nowadays, Xcode 15 is required for publishing apps on the App Store